extensions/blocks/calendly/edit.js

```diff
@@ -0,0 +1,367 @@
+		src += '&amp;text_color=' + textColor;
+		return src;
+	};
+
+	const inlinePreview = (
+		<>
+			<div className={ `${ className }-overlay` }></div>
+			<iframe
+				src={ iframeSrc() }
+				width="100%"
+				height="100%"
+				frameborder="0"
+				data-origwidth="100%"
+				data-origheight="100%"
+				style={ { minWidth: '320px', height: '630px', width: '100%' } }
+				title="Calendly"
+			></iframe>
+		</>
+	);
+
+	const submitButtonPreview = (
+		<SubmitButton
+			submitButtonText={ submitButtonText }
+			attributes={ attributes }
+			setAttributes={ setAttributes }
+		/>
+	);
+
+	const linkPreview = (
+		<>
+			<a style={ { alignSelf: 'flex-start', border: 'none' } } class="wp-block-button__link">
+				{ submitButtonText }
+			</a>
+		</>
+	);
+
+	const blockPreview = ( previewStyle, disabled ) => {
+		if ( previewStyle === 'inline' ) {
+			return inlinePreview;
+		}
+
+		if ( disabled ) {
+			return linkPreview;
+		}
+
+		return submitButtonPreview;
+	};
+
+	const styleOptions = [
+		{ value: 'inline', label: __( 'Inline', 'jetpack' ) },
+		{ value: 'link', label: __( 'Link', 'jetpack' ) },
+	];
+
+	const blockControls = (
+		<BlockControls>
+			{ url && (
+				<Toolbar
+					isCollapsed={ true }
+					icon="admin-appearance"
+					label={ __( 'Style', 'jetpck' ) }
+					controls={ styleOptions.map( styleOption => ( {
+						title: styleOption.label,
+						isActive: styleOption.value === style,
+						onClick: () => setAttributes( { style: styleOption.value } ),
+					} ) ) }
+				/>
+			) }
+		</BlockControls>
+	);
+
+	const inspectorControls = (
+		<InspectorControls>
+			{ url && (
+				<>
+					<PanelBody title={ __( 'Styles', 'jetpack' ) }>
+						<div className="block-editor-block-styles">
+							{ styleOptions.map( styleOption => {
+								return (
+									<div
+										key={ styleOption.value }
+										className={ classnames( 'block-editor-block-styles__item is-calendly', {
+											'is-active': styleOption.value === style,
+										} ) }
+										onClick={ () => {
+											setAttributes( { style: styleOption.value } );
+										} }
+										onKeyDown={ event => {
+											if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+												event.preventDefault();
+												setAttributes( { style: styleOption.value } );
+											}
+										} }
+										role="button"
+										tabIndex="0"
+										aria-label={ styleOption.label }
+									>
+										<div className="block-editor-block-styles__item-preview editor-styles-wrapper is-calendly">
+											{ blockPreview( styleOption.value, true ) }
+										</div>
+										<div className="block-editor-block-styles__item-label">
+											{ styleOption.label }
+										</div>
+									</div>
+								);
+							} ) }
+						</div>
+					</PanelBody>
+
+					<PanelColorSettings
+						title={ __( 'Embed Color Settings', 'jetpack' ) }
+						colorSettings={ [
+							{
+								value: '#' + backgroundColor,
```

Comment:
Same comment as above re consistent use of template literals versus string concat.